tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1180 @@
+              'cmr-hits': '1'
+            }
+          })
+        }
+      })
+
+      // Intercept granules timeline request
+      await page.route('**/search/granules/timeline', async (route) => {
+        const request = route.request()
+        const body = request.postData()
+        if (body) {
+          expect(body).toBe('end_date=2023-12-01T00:00:00.000Z&interval=day&start_date=2018-12-01T00:00:00.000Z&concept_id[]=C194001210-LPDAAC_ECS')
+        }
+
+        await route.fulfill({
+          body: JSON.stringify(projectGranuleTimelineBody),
+          headers: projectGranuleTimelineHeaders
+        })
+      })
+
+      // Intercept GraphQL request
+      await page.route('**/api', async (route) => {
+        const request = route.request()
+        const body = JSON.stringify(request.postData())
+        const expectedBody = JSON.stringify(graphQlGetCollection(conceptId))
+
+        const requestBody = JSON.parse(body)
+        const expectedRequestBody = JSON.parse(expectedBody)
+
+        if (JSON.stringify(requestBody) === JSON.stringify(expectedRequestBody)) {
+          await route.fulfill({
+            body: JSON.stringify(projectGranuleCollectionGraphQlBody),
+            headers: projectGranuleGraphQlHeaders
+          })
+        } else if (JSON.stringify(requestBody) === JSON.stringify(graphQlGetCollections('C194001210-LPDAAC_ECS'))) {
+          await route.fulfill({
+            body: JSON.stringify(projectGranuleCollectionsGraphQlBody),
+            headers: projectGranuleGraphQlHeaders
+          })
+        }
+      })
+
+      // Go to collection in the project context
+      await page.goto('/search/granules?p=C194001210-LPDAAC_ECS!C194001210-LPDAAC_ECS&pg[1][a]=2058417402!LPDAAC_ECS')
+
+      // Ensure the correct number of results were loaded
+      await testResultsSize(page, cmrHits)
+
+      const removeButtonLocator = page.getByTestId('granule-results-actions__proj-action--remove')
+      await removeButtonLocator.waitFor({ state: 'visible' })
+      await expect(removeButtonLocator).toBeVisible()
+
+      const downloadAllButtonLocator = page
+        .getByTestId('granule-results-actions__download-all-button')
+        .locator('.button__badge')
+      await downloadAllButtonLocator.waitFor({ state: 'visible' })
+    })
+  })
+
+  test.describe('When the path is loaded with a project collection', () => {
+    test('loads with all granules in the project', async ({ page }) => {
+      const cmrHits = 15073
+
+      await page.route('**/search/granules.json', (route) => {
+        const request = route.request()
+        const body = request.postData()
+        expect(body).toBe('echo_collection_id=C1214470488-ASF&page_num=1&page_size=20')
+        route.fulfill({
+          body: JSON.stringify(noParamsGranulesBody),
+          headers: {
+            ...commonGranulesHeaders,
+            'access-control-expose-headers': 'cmr-hits',
+            'cmr-hits': cmrHits
+          }
+        })
+      })
+
```

Comment:
```suggestion
 await page.route(/saved_access_configs/, async (route) => {
        await route.fulfill({
          json: {}
        })
      })
```
This is the other one where we were mocking this request on cypress same I saw the banner and then added this locally to make it go away